### ISSUE TITLE
FileNotFoundError

### ISSUE THREAD

**Issue by marzique** [bureaucratic-labs/dostoevsky#38]
`pip3 install dostoevsky`
then
`dostoevsky download vk-embeddings cnn-social-network-model`
which lead to error:
```
Traceback (most recent call last):
  File "/home/tarn/.local/bin/dostoevsky", line 22, in <module>
    downloader.download(source=source, destination=destination)
  File "/home/tarn/.local/lib/python3.6/site-packages/dostoevsky/data/__init__.py", line 29, in download
    with open(destination_path, 'wb') as output:
FileNotFoundError: [Errno 2] No such file or directory: '/home/tarn/.local/lib/python3.6/site-packages/dostoevsky/data/embeddings/vk-min-100-300d-none.tar.xz'
```

Ubuntu 18.04.2 LTS

**Comment by iamilyasedunov** [comment]
I had same question my problem were solved by pip3 install dostoevsky==0.1.2

**Comment by marzique** [comment]
Yeah thanks it worked, but immediately got few more, gonna close this one and open onether one

**Comment by dveselov** [comment]
Looks like packaging error, thanks for report.

**Comment by dveselov** [comment]
Should be fixed in `0.2.1`.  
Try `pip install -U dostoevsky==0.2.1`
